suite-native/transactions/src/components/TransactionDetail/TransactionDetailInputsSheetSection.tsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+
+import { A, G } from '@mobily/ts-belt';
+
+import { Icon } from '@suite-common/icons';
+import { NetworkSymbol, isNetworkSymbol } from '@suite-common/wallet-config';
+import { TokenSymbol } from '@suite-common/wallet-types';
+import { Box, Card, Text } from '@suite-native/atoms';
+import {
+    AccountAddressFormatter,
+    CryptoAmountFormatter,
+    EthereumTokenAmountFormatter,
+} from '@suite-native/formatters';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { TransactionTranfer } from '../../selectors';
+
+type TransactionAddressAmountProps = {
+    address: string;
+    amount?: string;
+    symbol: NetworkSymbol | TokenSymbol;
+    decimals?: number;
+};
+
+const addressAmountColumnStyle = prepareNativeStyle(_ => ({
+    maxWidth: '42.5%',
+}));
+
+const TransactionAddressAmount = ({
+    address,
+    amount,
+    symbol,
+    decimals,
+}: TransactionAddressAmountProps) => (
+    <Box>
+        <AccountAddressFormatter value={address} variant="hint" />
+        {amount &&
```

Comment:
Guess you could just return null if amount is nullable? Or is there a case where you want to render just address without amount?

Reply:
It is not possible. We want to render this component even without the amount in some cases. See the demo screenshots attached.